README.md

```diff
@@ -1,8 +1,151 @@
-# sindri-scroll-sdk
+# Sindri Provers for Scroll SDK
 
-## Temporary Usage Instructions
 
-First, create your `config.json` file from a template
+<img src="./media/sindri-gradient-logo.webp" height="160" align="right"/>
+
+#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/) | [Getting Started](#getting-started) | [Development](#development)
+
+Sindri provides automated ZK proving infrastructure, empowering hundreds of teams — including leading Layer 2s and rollups — to launch in minutes instead of months.
+Through our API, developers can seamlessly integrate verifiable computation, reducing time to market, cutting costs, and scaling faster.
+Sindri makes zero-knowledge infrastructure simple and accessible, facilitating automation across every layer of the ZK app deployment stack.
+
+This repository hosts the images and Helm charts designed to perform proof generation for a Scroll SDK chain.
```

Comment:
```suggestion
This repository hosts images and Helm charts designed to perform proof generation for a Scroll SDK chain.
```

Reply:
Also, perhaps define the type of images hosted?

Reply:
Commit https://github.com/Sindri-Labs/sindri-scroll-sdk/pull/8/commits/9be1a2b57440f1ba59d28d94d3d7b75f408e822c changed the sentence to:
> This repository hosts a proving application image and Helm chart which will route all zkEVM proof generation to the Sindri platform for a Scroll SDK chain.

(Also lumping in a fix for the fact that we are only hosting one image and Helm chart, so those should both be singular)